Library/Homebrew/cmd/update.rb

```diff
@@ -131,6 +133,10 @@ def update
     Descriptions.update_cache(report)
   end
 
+  def shortenRevision(revision)
```

Comment:
Oops, it's been a long day... addressed in https://github.com/Homebrew/homebrew/commit/3e7809b3c91defe089fcfc6b5913ee7183b32a12